articles/how-to-implement-mlops/index.md

```diff
@@ -0,0 +1,55 @@
+
+Read more about DevOps engineering [here](https://www.section.io/engineering-education/what-it-takes-to-be-a-devops-engineer/).
+
+MLOps is a [compound](https://caiomsouza.medium.com/mlops-machine-learning-and-operations-and-ai-at-scale-ffcac7e50f62) of machine learning (ML) and operations (Ops). With this practice, operations professionals and data scientists collaborate and communicate better to manage [production machine learning](https://towardsdatascience.com/production-machine-learning-isnt-hard-anymore-932bd91e138f) or [machine learning life cycle](https://www.educba.com/machine-learning-life-cycle/#:~:text=Machine%20Learning%20Life%20Cycle%20is,are%20involved%20in%20various%20applications). Essentially, MLOps comes in to manage the deployment of deep learning and machine learning models in large-scale production environments.
+
+MLOps includes a data science team for curating datasets and building AI models that analyze the datasets. Machine learning engineers are also part of the MLOps team, and they have the role of running datasets through the models in automated ways.
+
+There exist three levels of MLOps, as [described](https://cloud.google.com/solutions/machine-learning/mlops-continuous-delivery-and-automation-pipelines-in-machine-learning) by Google: MLOps level 0, MLOps level 1, and MLOps level 2. The choice between each level is dependent on an organization&#39;s size and the number of machine learning algorithms they target to run.
+
+- **MLOps level 0 -** This means thatan organization recognizes the need to adopt machine learning practices and have an in-house data scientist capable of deploying and building models. However, ML workflow in that entity remains mainly manual.
+
+- **MLOps level 1 -** This involves machine learning pipeline automation whereby the organization introduces a pipeline to enhance continuous training. Model and data validation is entirely automatic. A reduction in model performance or the introduction of new data triggers retraining. Solutions operating in a constantly changing environment better use this scenario to respond proactively to shifts in price rates, customer behavior, or other factors.
+
+- **MLOps level 2 –** With this level, [CI/CD pipeline](https://semaphoreci.com/blog/cicd-pipeline#:~:text=What%20is%20a%20CI%2FCD,and%20enable%20fast%20product%20iterations.) automates the deployment of machine learning models and elements of machine learning [training pipelines in production](https://www.altexsoft.com/blog/machine-learning-pipeline/). Tech-driven organizations can rely on this level for hourly or daily retraining of models, updating models in minutes, and simultaneous redeployment of these models on multiple servers. Such organizations require an end-to-end MLOps cycle for them to survive.
```

Comment:
A gentle one-line introduction to CI/CD pipeline would be better for covering a broader audience.